Ethereal Artefacts/Ethereal Artefacts/Views/HomepageView.swift

```diff
@@ -7,14 +7,18 @@
 
 import SwiftUI
 
-struct HomepageView: View {    
+struct HomepageView: View {
+    @State private var searchText = ""
+    
     var body: some View {
-        NavigationView() {
+        NavigationStack() {
             VStack {
                 HStack {
                     Text("Home")
                         .font(.title)
                         .fontWeight(.bold)
+                        .searchable(text: $searchText)
```

Comment:
You can play around with the native Navigation Bar. Check out the `.navigationBarTitle`, `.navigationBarTitleDisplayMode` and `.toolbar` modifiers. Those are usually placed on the outermost `Vstack` in the struct's hierarchy and allow you to achieve this Navigation Bar behavior completely natively.

Note: The `.searchable` modifier can then be chained on top of those (yet again, on the main `VStack`) but you can do this at a later point in time.

Reply:
Done.